completions/bts

```diff
@@ -1,22 +1,42 @@
 # bts completion                                      -*- shell-script -*-
 
 # List bug numbers from bugs cache in ~/.devscripts_cache/bts
-# TODO:API: rename per conventions, rework to use vars rather than outputting
+# @deprecated 2.12 Use `_comp_cmd_bts__cached_bugs`
```

Comment:
Similarly here, I don't think we need to keep the old versions.

Reply:
Removed.